dydx-stark/src/endpoint/send.ts

```diff
@@ -0,0 +1,51 @@
+import objectPath from 'object-path'
+import { ExecuteWithConfig } from '@chainlink/types'
+import { Requester, Validator, logger } from '@chainlink/external-adapter'
+import { Config, DEFAULT_DATA_PATH } from '../config'
+import { PriceDataPoint, requireNormalizedPrice, getPricePayload } from './starkex'
+
+export const NAME = 'send'
+
+const customParams = {
+  dataPath: false,
+  result: false,
+  asset: true,
+}
+
+export const execute: ExecuteWithConfig<Config> = async (request, config) => {
+  const validator = new Validator(request, customParams)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const { asset, ...data } = validator.validated.data
+
+  const dataPath = data.dataPath || DEFAULT_DATA_PATH
+  const price = <number | string>objectPath.get(data, dataPath)
```

Comment:
Why do we parse the dataPath here, instead of using the copy adapter in the task pipeline in the job?

Reply:
Just so there is another option for example when using this from another composite adapter. By default, it'll look in the `data.result` so it should not need a copy adapter or a specific `dataPath` set, by default.

Reply:
Ok. Seems like it's a bit premature feature (haven't seen the need for it before as most adapters pre-set the result key, but maybe I just don't understand the usecase), but looks good otherwise.